paper/paper.md

```diff
@@ -73,26 +73,40 @@ of the global solar energy industry demands correspondingly more
 capable models.  Per the United States Department of Energy,
 "the importance of accurate modeling is hard to overstate" [@seto2022].
 
-Compared with other PV modeling tools, pvlib python stands out in several
+Compared with other modern PV system modeling tools, pvlib python stands out in several
 key aspects.  One is its toolbox design, providing the user a
 level of flexibility and customization beyond that of other tools.  Rather than organizing
-the user interface around pre-built modeling workflows, pvlib python
-makes the individual "building blocks" of PV performance models accessible to
+the user interface around pre-built modeling workflows as in other PV system
+modeling tools (e.g. SAM [@Gilman2018], PVsyst [@Mermoud1994], SolarFarmer [@Mikofski2018],
+PlantPredict [@Passow2017], and CASSYS [@Pai2016], to name a few software tools with
+comparable breadth of PV system modeling capability), pvlib python
+makes the individual "building blocks" of PV system performance models accessible to
 the user.  This allows the user to assemble their own model workflows, including
 the ability of incorporating custom modeling steps.  This flexibility
-is essential for applications in both academia and industry.
+is essential for applications in both academia and industry.  To our knowledge,
+the only other PV system modeling software with such a toolbox design is the
+MATLAB version of pvlib [@Andrews2014], the code base from which pvlib python
+was translated and has since surpassed in capability and community uptake.  
```

Comment:
This could sound like pvlib python has been overtaken - could probably be written more clearly.

Can we say that Matlab version is no longer being actively developed?